tlair/src/main/scala/at/forsyte/apalache/tla/lir/io/CounterexampleWriter.scala

```diff
@@ -57,38 +57,41 @@ class TlaCounterexampleWriter(writer: PrintWriter) extends CounterexampleWriter
     if (state.isEmpty) {
       pretty.write(ValEx(TlaBool(true)))
     } else {
-      state.toList.sortBy(_._1).foreach{
-        case (name,value) =>
-          writer.print(s"/\\ $name = " )
-          pretty.write(value)
-          writer.println
+      state.toList.sortBy(_._1).foreach { case (name, value) =>
+        writer.print(s"/\\ $name = ")
+        pretty.write(value)
+        writer.println
       }
     }
   }
 
-  override def write(rootModule: TlaModule, notInvariant: NotInvariant, states: List[NextState] ) : Unit = {
+  override def write(rootModule: TlaModule, notInvariant: NotInvariant, states: List[NextState]): Unit = {
     val pretty = new PrettyWriter(writer)
 
     writer.println("%s MODULE counterexample %s\n".format("-" * 25, "-" * 25))
     writer.println("EXTENDS %s\n".format(rootModule.name))
 
     states.zipWithIndex.foreach {
+      case (state, 0) =>
+        writer.println("(* Constant initialization state *)")
+        writer.println(s"\nStateConst ==")
```

Comment:
```suggestion
        writer.println(s"\nConstInit ==")
```

Reply:
We usually call a constant initializer `ConstInit` in the specs and the manual.

Reply:
I copied the name from your comment in #265, but that's an easy fix.